src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAX_BALL_NUMBER = 100;
```

Comment:
How do you think what max number will be returned by random.nextInt(MAX_BALL_NUMBER), where MAX_BALL_NUMBER = 100?